group.go

```diff
@@ -86,6 +86,19 @@ func (g *group) OPTIONS(route string, hs ...common.Handler) {
 	g.r.OPTIONS(route, newHandler(hs))
 }
 
+// Handle will create a route for any method
+func (g *group) Handle(method, route string, hs ...common.Handler) {
```

Comment:
Should we add tests?

Reply:
Hmm! Maybe when we do some wholesale changes on the testing front. I'd like to get more granular testing implemented in general